### ISSUE TITLE
AZ "Connect to Peer" has to be selected 2 times before peers are obtained

### ISSUE THREAD

**Issue by ethers** [ethereum/cpp-ethereum#1552]
develop e212074

Run AZ.  Network->Connect to Peer has to be selected 2 times before peers are obtained, otherwise the log is stuck at:
```
*N* [ 02:09:30 PM | p2p ] p2p.started id: 44348e6f…
```

**Comment by ethers** [comment]
saw this couple of times but most recently a single connectToPeer worked, so closing for now

**Comment by ethers** [comment]
Saw this again.  Doing Network->Connect to Peer single time doesn't connect to any peers.
```
--- [ 03:58:47 PM | main ] startWorking for thread p2p
--- [ 03:58:47 PM | main ] Spawning p2p
--- [ 03:58:47 PM | p2p ] startWorking for thread ethsync
--- [ 03:58:47 PM | p2p ] Spawning ethsync
--- [ 03:58:47 PM | p2p ] startWorking for thread anon
--- [ 03:58:47 PM | p2p ] Spawning anon
*** [ 03:58:55 PM | p2p ] UPnP device not found.
+N+ [ 03:58:55 PM | p2p ] Listening on local port 30305  (public: 0.0.0.0:0 )
+P+ [ 03:58:55 PM | p2p ] refreshing buckets
+P+ [ 03:58:55 PM | p2p ] Terminating discover after 0  rounds.
*N* [ 03:58:55 PM | p2p ] p2p.started id: 44348e6f…
+P+ [ 03:58:55 PM | p2p ] Terminating discover after 0  rounds.
+P+ [ 03:59:25 PM | p2p ] Terminating discover after 0  rounds.
+P+ [ 03:59:55 PM | p2p ] Terminating discover after 0  rounds.
+P+ [ 04:01:23 PM | p2p ] Terminating discover after 0  rounds.
+P+ [ 04:01:23 PM | p2p ] refreshing buckets
```

here's the small bt in case it helps:
```
#0  0x00007f63375da12d in poll () at ../sysdeps/unix/syscall-template.S:81
#1  0x00007f6336448fe4 in ?? () from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#2  0x00007f63364490ec in g_main_context_iteration ()
   from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#3  0x00007f633db4a5e4 in QEventDispatcherGlib::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) () from /opt/Qt54/5.4/gcc_64/lib/libQt5Core.so.5
#4  0x00007f633daee04b in QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) () from /opt/Qt54/5.4/gcc_64/lib/libQt5Core.so.5
#5  0x00007f633daf2f65 in QCoreApplication::exec() ()
   from /opt/Qt54/5.4/gcc_64/lib/libQt5Core.so.5
#6  0x00000000006509d8 in main (argc=1, argv=0x7fff253f1e88)
    at /home/user/cpp-ethereum/alethzero/main.cpp:11
```

**Comment by subtly** [comment]
What peer are you trying to connect to?  Can you increase log level to 10? (drag slider on right side in AZ, or from CLI, -v 10)

**Comment by subtly** [comment]
Ah -- also, please retry. This is likely to have been fixed by #1563.

**Comment by ethers** [comment]
have not seen with the newer builds in past 2 days so closing.
